test/src/SpendPermissions/revoke.t.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+import {Test, console2} from "forge-std/Test.sol";
+
+import {SpendPermissions} from "../../../src/SpendPermissions.sol";
+
+import {SpendPermissionsBase} from "../../base/SpendPermissionsBase.sol";
+
+contract RevokeTest is Test, SpendPermissionsBase {
+    function setUp() public {
+        _initializeSpendPermissions();
+    }
+
+    function test_revoke_revert_invalidSender() public {}
+    function test_revoke_success() public {}
```

Comment:
ditto on tests to describe the effects in success case
- `test_revoke_success_isNotAuthorized`
- `test_revoke_success_emitsEvent`